package.json

```diff
@@ -10,6 +10,7 @@
     "dev": "nodemon src/index.ts",
     "start": "node build/src/index.js",
     "start:test": "dotenv -e test/.env.test yarn dev",
+    "test:setup": "./test/setup.sh",
```

Comment:
what is the process of running it? Should I define some variables?
<img width="663" alt="Screenshot 2023-10-10 at 11 56 45" src="https://github.com/snapshot-labs/snapshot-sequencer/assets/7693369/cddab249-6fce-4871-8b6d-06d07deee312">

Reply:
Is your mysql database running locally ? Do you have the `mysql` cli installed ?

Reply:
> yes, it runs locally. No, I don't have `mysql` cli - maybe we should add it to devDependencies if it's required?

The `mysql` cli should be shipped with the mysql install, and can not be installed with npm